multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/resources/ConfigurationEntriesResource.java

```diff
@@ -37,7 +37,7 @@
     @PostMapping("/purge")
     public ResponseEntity<Void> purgeConfigurationRegistry(@RequestParam(REQUEST_PARAM_ORGANIZATION) String organization,
                                                            @RequestParam(REQUEST_PARAM_SPACE) String space) {
-        CloudControllerClient client = createClient(organization, space);
+        CloudControllerClient client = createClient();
```

Comment:
Why you don't use organization and space to create client?

Reply:
Firstly because I wanted to remove the factory method for (user, org, space). It is used only here. Secondly getting client for the concrete space is not needed here, as the spaceid is retrieved again later when purging the config entries. Thirdly, one call to the CC could be skipped here, because creating client with org and space name involves additional call to obtain the space guid.